backend/requirements.txt

```diff
@@ -41,3 +41,9 @@ uritemplate==3.0.1
 urllib3==1.25.8
 Werkzeug==1.0.0
 wrapt==1.11.2
+pandas == 1.0.1
+numpy == 1.18.4
+django-pandas== 0.6.1
+pandas == 1.0.1
+numpy == 1.18.4
+django-pandas== 0.6.1
```

Comment:
You're importing the same packages twice here.

Reply:
Yes you are right. I will re push it.